### DISCUSSION TITLE
Persisted cache for pwa apps

### DISCUSSION THREAD

**Discussion by sharmapukar217** [blitz-js/blitz#3918] [General] [answered]
Is there any way to cache the data because I'm trying to make a pwa app and all the rpc calls seems to be POST requests that couldn't cached by service worker

**Comment by siddhsuresh** [comment, marked as answer]
> Is there any way to cache the data because I'm trying to make a pwa app and all the rpc calls seems to be POST requests that couldn't cached by service worker

Hey @sharmapukar217, support for RPC `GET`  requests is currently in development/review at https://github.com/blitz-js/blitz/pull/3891

**Reply by sharmapukar217** [reply]
It would be more nicer if we could use react query's persistor to cache data. It this possible? I tired and its caching the data but couldn't retrieve and shows error
